gem/views.py

```diff
@@ -145,9 +146,12 @@ def clean_comment(self):
         keyword = keyword.replace('\r', '')
         match = re.search(keyword.lower(), comment.lower())
```

Comment:
@smn @miltontony How do you guys feel about `keyword.lower()`? I'm a bit concerned as this will change the meaning of regexes with special sequences (eg: \D). I'd be more comfortable not lowering the keyword and instructing users to write keywords as lowercase in the help text for the field on the GemSettings model.